src/views/preview/comment/comment-container.jsx

```diff
@@ -0,0 +1,109 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const api = require('../../../lib/api');
+const log = require('../../../lib/log');
+const bindAll = require('lodash.bindall');
+const classNames = require('classnames');
+
+const FlexRow = require('../../../components/flex-row/flex-row.jsx');
+const Comment = require('./comment.jsx');
+
+require('./comment.scss');
+
+class CommentContainer extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleExpandThread'
+        ]);
+        this.state = {
+            expanded: false,
+            replies: []
+        };
+    }
+
+    componentDidMount () {
+        this.fetchReplies(this.props.id);
+    }
+
+    fetchReplies (id) {
+        if (this.props.reply_count > 0) {
+            api({
+                uri: `/comments/project/${this.props.projectId}/${id}`
+            }, (err, body) => {
+                if (err) {
+                    log.error(`Error fetching comment replies: ${err}`);
+                    return;
+                }
+                if (typeof body === 'undefined') {
+                    log.error('No comment reply information');
+                    return;
+                }
+                this.setState({
+                    replies: body
+                });
+            });
+        }
+    }
+
+    handleExpandThread () {
+        this.setState({
+            expanded: true
+        });
+    }
+
+    render () {
+        const {
+            author,
+            content,
+            datetime_created,
```

Comment:
You could argue we shouldn't be sending this down in the JSON, but would you please convert this to camelCase?